hr_timesheet_sheet_no_negative/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.hr.timesheet.no.negative</field>
```

Comment:
You don't need this long (and difficult to read) line. Better let the name be generated automatically by the system IMO.

Reply:
😂 that's just tastes

Reply:
The generated automatically is shorter?

Reply:
removed, but I left it commented to avoid anybody to fill it with another name